src/index.ts

```diff
@@ -1,117 +1,129 @@
+					Utils.streamToBuffer(stream).then((buffer: Buffer) => {
+						// send the file along to the next tar stream regardless
+						pack.entry(header, buffer);
+
+						// create a FileInfo from the header
+						const info: FileInfo = {
+							name: Utils.normalizeTarEntry(header.name),
+							size: header.size,
+							contents: buffer
+						};
+
+						// Now provide the resolvers with the information and file
+						resolvers.map(resolver => {
+							resolver.entry(info);
+						});
+						next();
+					});
 				}
+			);
 
-				// Now provide the resolvers with the information and file
-				resolvers.map((resolver) => {
-					resolver.entry(info)
-				})
-				next()
-			})
-		})
-
-		extract.on('finish', () => {
-
-			let maybeResolver = _(resolvers)
-				.orderBy((val: Resolver) => val.priority, ['desc'])
-				.find((resolver: Resolver) => resolver.isSatisfied(bundle))
-
-			// if no resolver was happy this is an error
-			if (maybeResolver === undefined) {
-				reject(new Error('No project type resolution could be performed'))
-				return
-			}
-
-			const resolver: Resolver = maybeResolver
-
-			// Now that we have a resolver, add the new files needed to the stream
-			resolver.resolve(bundle)
-			.then((additionalItems) => {
-
-				return Promise.map(additionalItems, (file: FileInfo) => {
-					pack.entry({ name: file.name, size: file.size }, file.contents)
-					if (file.name === 'Dockerfile') {
-						return bundle.callDockerfileHook(file.contents.toString())
-					}
-				})
-				.then(() => {
+			extract.on('finish', () => {
+				let maybeResolver = _(resolvers)
+					.orderBy((val: Resolver) => val.priority, ['desc'])
+					.find((resolver: Resolver) => resolver.isSatisfied(bundle));
 
-					return Promise.try(() => {
-						if (resolver.name === 'Standard Dockerfile') {
-							// The hook will not have been ran on this file yet, as the Dockerfile was not added
-							// as an additional item
-							return bundle.callDockerfileHook((resolver as DockerfileResolver).getDockerfileContents())
-						}
-					})
-
-				})
-				.then(() => {
-
-					// all of the extra files have now been added to the stream, resolve
-					// the promise with it
-					pack.finalize()
+				// if no resolver was happy this is an error
+				if (maybeResolver === undefined) {
+					reject(new Error('No project type resolution could be performed'));
+					return;
+				}
 
-					resolve({
-						projectType: resolver.name,
-						tarStream: pack
+				const resolver: Resolver = maybeResolver;
+
+				// Now that we have a resolver, add the new files needed to the stream
+				resolver
+					.resolve(bundle)
+					.then(additionalItems => {
+						return Promise.map(additionalItems, (file: FileInfo) => {
+							pack.entry({ name: file.name, size: file.size }, file.contents);
+							if (file.name === 'Dockerfile') {
+								return bundle.callDockerfileHook(file.contents.toString());
+							}
+						})
+							.then(() => {
```

Comment:
Yeah it just seems like a very arbitrary indentation for no reason, the other changes prettier makes have a clear reason, whether or not they fit your preference